src/libs/actions/Policy.ts

```diff
@@ -1108,16 +1165,16 @@ function buildOptimisticCustomUnits() {
 /**
  * Optimistically creates a Policy Draft for a new workspace
  *
- * @param {String} [policyOwnerEmail] Optional, the email of the account to make the owner of the policy
- * @param {String} [policyName] Optional, custom policy name we will use for created workspace
- * @param {String} [policyID] Optional, custom policy id we will use for created workspace
- * @param {Boolean} [makeMeAdmin] Optional, leave the calling account as an admin on the policy
+ * @param [policyOwnerEmail] Optional, the email of the account to make the owner of the policy
+ * @param [policyName] Optional, custom policy name we will use for created workspace
+ * @param [policyID] Optional, custom policy id we will use for created workspace
+ * @param [makeMeAdmin] Optional, leave the calling account as an admin on the policy
```

Comment:
Maybe we can remove "Optional" from comments here now, since it's obvious from types